pdal/Options.hpp

```diff
@@ -63,15 +63,17 @@ class PDAL_DLL Option
     template <typename T>
     Option(std::string const& name, const T& value) : m_name(name)
     {
-        std::ostringstream oss;
-        oss << value;
-        m_value = oss.str();
+        m_value = Utils::toString(value);
     }
 
-    Option(std::string const& name, const std::string& value) :
-        m_name(name), m_value(value)
+    Option(std::string const& name, const std::string& value) : m_name(name), m_value(value)
     {}
 
+    Option(std::string const& name, const double& value) : m_name(name)
+    {
+        m_value = Utils::toString(value, 15);
```

Comment:
What about providing a specialization ctor that 

```
Option(std::string const& name, 
              const double& value, 
              double precision=15)
```

Reply:
I don't think there's any practical advantage.

Reply:
The advantage is the ability for a user of Option to control it in the few cases it might matter, like the one that precipitated this change.

Reply:
I don't think there's a good argument for less precision.

Reply:
I agree. Is there an argument for *more* precision, however?

Reply:
There may be, but I doubt a user will know this but it's probably better to avoid transit in strings if this isn't sufficient.